src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -62,6 +62,13 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('no_push', help="Indicates whether the image built should be pushed to the registry.", arg_type=get_three_state_flag())
         c.argument('no_logs', help="Do not show logs after successfully queuing the build.", action='store_true')
 
+    with self.argument_context('acr import') as c:
+        c.argument('source', help="The source identifier in the format '[registry.azurecr.io/]repository:tag' or '[registry.azurecr.io/]repository@digest'.")
```

Comment:
`'[registry.azurecr.io/]repository[:tag]'`

Reply:
I do not think tag is optional for source...unless you want to import a collection of images rather than a single one, do you?

Reply:
Could you please validate? Think doesn't help here ...

Reply:
FYI https://github.com/Azure/azure-rest-api-specs/blob/master/specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json#L1117

Reply:
Cool

Reply:
What if somebody is so unlucky that there is no latest tag in that repository he/she specifies?

Reply:
It is possible right? Like all the images either built or imported into the repository are not defaulted to latest when they started to exist in the repository.

Reply:
How am I supposed to deal with it?

Reply:
Optionalized the tag